src/wallet/rpc/backup.cpp

```diff
@@ -1383,6 +1388,8 @@ RPCHelpMan importmulti()
                 nLowestTimestamp = timestamp;
             }
         }
+
+        EnsureBlockDataFromTime(pwallet->chain(), nLowestTimestamp);
```

Comment:
In 46b7f731b1e98b0836c5b97cd99c0887d6173a17 "rpc: Enable wallet import on pruned nodes and add test"

We don't want to do this for `importmulti`. `importmulti` can have some imports succeed, and others fail. Just because one import has an out of range timestamp does not mean the entire import should fail the RPC. This check would result in a JSONRPCError which is not what we expect from `importmulti` - rather there should be an error per import that has a too old timestamp.

Furthermore, we already handle the pruned node case. If the rescan reports a time that is more recent than the timestamp for a given import, we will give an error for that import with more details about what happened, maybe why it happened, and how the user can resolve it.